### DISCUSSION TITLE
Inquietudes construcción del data set

### DISCUSSION THREAD

**Discussion by joanfco30** [rramosp/ai4eng.v1#119] [04 - GESTIÓN DE DATOS]
Buenas tardes un cordial saludo. 
Realizando este laboratorio me encuentro bastante perdido al momento de construir. construyo el data set con la información solicitada y aquí me siento bloqueado para continuar. 
Cualquier comentario me sería  de gran ayuda, muchas gracias. 

![image](https://user-images.githubusercontent.com/60493083/163732695-0797d380-27ca-4d17-920b-5203b9233fe5.png)

**Comment by rramosp** [comment]
hola joanfco, cuál es el laboratorio que estás haciendo??

On Sun, Apr 17, 2022 at 4:34 PM joanfco30 ***@***.***> wrote:

> Buenas tardes un cordial saludo.
> Realizando este laboratorio me encuentro bastante perdido al momento de
> construir. construyo el data set con la información solicitada y aquí me
> siento bloqueado para continuar.
> Cualquier comentario me sería de gran ayuda, muchas gracias.
>
> [image: image]
> <https://user-images.githubusercontent.com/60493083/163732695-0797d380-27ca-4d17-920b-5203b9233fe5.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/119>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIN76GG3NPLJDGBF3ELVFR7YBANCNFSM5TUNLDPQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Reply by joanfco30** [reply]
Buenos días, un cordial saludo. Es el llamado LAB 04.02 - BUILDING DATASETS. 
La primera parte es construir un data set con los siguientes datos: 
![image](https://user-images.githubusercontent.com/60493083/163814057-d3f6ea7c-c167-4c7c-9fc9-1e34f32afec2.png)
Yo realizo la construcción del data set de la siguiente manera: 
![image](https://user-images.githubusercontent.com/60493083/163814176-2a8f6719-e952-4dfa-81a8-2e8a2f1fc30c.png)
 En esta etapa es donde me siento perdido y no se como avanzar. 
 Muchas gracias y quedo atento a cualquier comentario o ayuda para continuar este laboratorio.
